src/om_bootstrap/nav.cljs

```diff
@@ -131,10 +131,10 @@
 (defn render-toggle-button [owner bs]
   (let [handle-toggle (fn []
                         (when-let [f (:on-toggle bs)]
-                          (om/set-state-nr! owner [:changing?] true)
```

Comment:
can you explain this change? I didn't want to trigger a render because this change is triggered BY a re-render.